### ISSUE TITLE
[Suggestion] Chests tweaks

### ISSUE THREAD

**Issue by ghost** [asiekierka/CharsetMC#263]
I'm enjoying the chests mod, and I'd like to make three suggestions:

- [ ] 1. Reinforced chests. Just add some metal hardware to double the capacity. ~~(Better Storage/BS Too had reinforced chests but they're no longer being updated.)~~
- [X] 2. ~~Cabinets like rustic. So much like chests. They just want a little vertical love. :)~~ (Rustic has any plank cabinets now.)
- [ ] 3. ~~Chest materials JSON. Rather than run around coding every single wood, let users add materials and use the texture to create the model. Bonus - TNT chests, etc. :D~~ (Got some sleep. ;) Going to try using CraftTweaker or something to trick Charset into using non-conforming woods.)

Thank you.

Edit: Better Storage was taken over by Better Storage Too. It is being developed, but hasn't been released on Curse yet. (Plus no where near the mod support Charset Chests provides.)

**Comment by asiekierka** [comment]
* With regards to reinforced chests, that is not planned. I am considering a facility to allow defining custom chest materials with custom container sizes, but I'm not sure yet.
* Cabinets may be added, but I'm not 100% sure. I don't like redundnacy.
* "run around coding every single wood" - They're detected automatically! Besides, you can add new materials - just write a new recipe .JSON, and add some new materials via CraftTweaker.

**Comment by copygirl** [comment]
> Cabinets may be added, but I'm not 100% sure. I don't like redundnacy.

Lockers were one of the most praised and used blocks in Better Storage. Along with the ability to make them out of any wood type, using that wood's texture, I think it would be a great addition players might like a lot.

**Comment by asiekierka** [comment]
Maybe...

**Comment by ghost** [comment]
The thing about the wood was because in-game it looks like you created textures for each type, but you probably just worked some graphics processing magic.

**Comment by asiekierka** [comment]
Yes. For proper support of metallic and stone-based materials, new base textures would have to be made. A way to override textures on a per-material basis is planned, but most likely not soon. (Not even sure if in 1.12.)

**Comment by ghost** [comment]
Hmm... For the first, I was just thinking of metal for the edging, maybe taken from blocks like you do with wood. The third was more an enthusiasm for being helpful. :) Now that I've actually slept, I think Bibliocraft has something that lets you frame odd materials for chests.

**Comment by asiekierka** [comment]
@scribbler957 As of 1.0.7, Rustic now has any-plank-type cabinets.

**Comment by ghost** [comment]
@asiekierka Sweet. :)
